hail/src/main/scala/is/hail/io/fs/GoogleStorageFS.scala

```diff
@@ -107,12 +107,11 @@ class GoogleStorageFS(serviceAccountKey: String) extends FS {
 
   @transient private var codecs: IndexedSeq[hadoop.io.compress.CompressionCodec] = _
 
-  private val defaultHadoopConf = new hadoop.conf.Configuration()
-
   def createCodecs(): Unit = {
     if (codecs != null)
       return
 
+    val defaultHadoopConf = new hadoop.conf.Configuration()
```

Comment:
what's this about?

Reply:
The codec interface had some legacy dependence on hadoop.  That was just ripped out by a separate PR, so this is gone and the GSFS is "clean" now.